packages/block-library/src/file/edit.js

```diff
@@ -120,7 +127,9 @@ function FileEdit( { attributes, isSelected, setAttributes, clientId } ) {
 			displayPreview: isPdf ? true : undefined,
 			previewHeight: isPdf ? 600 : undefined,
 			fileId: `wp-block-file--media-${ clientId }`,
+			blob: undefined,
 		} );
+		setTemporaryURL();
```

Comment:
I wonder if it might make sense to have a helper util for this

Reply:
I think there's a lot of duplication between how these media blocks are setup, a lot could be share but there are also some things that are to explain: some use "url" others "href" and others "src" for the uploaded files